2024-08-20 17:08:24.222616: W tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'libcudart.so.11.0'; dlerror: libcudart.so.11.0: cannot open shared object file: No such file or directory; LD_LIBRARY_PATH: /nfs/dust/cms/user/puschman/rivet/lib:/nfs/dust/cms/user/puschman/rivet/lib64:/nfs/dust/cms/user/puschman/rivet/lib64:/nfs/dust/cms/user/puschman/rivet/lib64:/nfs/dust/cms/user/puschman/rivet/lib64:/cvmfs/sft.cern.ch/lcg/app/releases/ROOT/6.32.02/x86_64-almalinux9.4-gcc114-opt/lib:/cvmfs/sft.cern.ch/lcg/releases/LCG_106_cuda/cuda/12.4/x86_64-el9-gcc11-opt/lib64/cmake:/cvmfs/sft.cern.ch/lcg/releases/LCG_106_cuda/cuda/12.4/x86_64-el9-gcc11-opt/lib64/stubs:/cvmfs/sft.cern.ch/lcg/releases/LCG_106_cuda/cuda/12.4/x86_64-el9-gcc11-opt/lib64:/cvmfs/sft.cern.ch/lcg/releases/LCG_106_cuda/cudnn/9.1.1.17/x86_64-el9-gcc11-opt/lib64:/cvmfs/sft.cern.ch/lcg/releases/LCG_106_cuda/cudnn/9.1.1.17/x86_64-el9-gcc11-opt/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/11.3.0-de683/x86_64-el9/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/11.3.0-de683/x86_64-el9/lib64:/cvmfs/sft.cern.ch/lcg/releases/binutils/2.40-acaab/x86_64-el9/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/11.3.0-de683/x86_64-el9/lib:/cvmfs/sft.cern.ch/lcg/releases/gcc/11.3.0-de683/x86_64-el9/lib64:/cvmfs/sft.cern.ch/lcg/releases/binutils/2.40-acaab/x86_64-el9/lib:/cvmfs/grid.cern.ch/alma9-ui-test/lib64:/cvmfs/grid.cern.ch/alma9-ui-test/lib:/cvmfs/grid.cern.ch/alma9-ui-test/usr/lib64:/cvmfs/grid.cern.ch/alma9-ui-test/usr/lib:/cvmfs/grid.cern.ch/alma9-ui-test/usr/lib64/dcap
2024-08-20 17:08:24.222819: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.
